### DISCUSSION TITLE
Date jump method

### DISCUSSION THREAD

**Discussion by rbmnex** [vkurko/calendar#220] [Q&A] [answered]
Hi,

There any way this calendar have method date jump like when I select date from datepicker and pass date jump method and the calendar displayed selceted date I pick?

**Comment by vkurko** [comment, marked as answer]
Hi, you need to update the [date](https://github.com/vkurko/calendar?tab=readme-ov-file#date) option with `ec.setOption('date', newDate)`.

**Reply by rbmnex** [reply]
thank you
